lib/local-storage-adapter.js

```diff
@@ -13,6 +14,12 @@ function createLocalStorageAdapter (queueName) {
   var backoffTimeKey = BACKOFF_TIME_KEY.replace('*', queueName)
   var errorCountKey = ERROR_COUNT_KEY.replace('*', queueName)
 
+  var cacheInSync = false
+  var storageInSync = true
+  var queueCache = []
+
+  var flushOnDefer = _.debounce(flush, 0)
```

Comment:
I was a little surprised by the debounce of 0, but this is just to avoid a write when we have a bunch of enqueues sequentially?

I've never actually looked into it, but from what I've observed we get a ton of events from jolt on a page load (3-10 depending on page content), but there occasionally appears to be a very small interval between them (or at least groups of them). Could it be worth debouncing by a slightly larger small amount of time (say 100-200ms) given we're debouncing anyway. Obviously this would introduce a greater risk of event loss, but could be worth it to speed up page loads more?